cmd/ingester/main.go

```diff
@@ -70,7 +70,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	chunkStore, err := chunk.NewStore(chunkStoreConfig, schemaConfig, storageClient)
+	chunkStore, err := chunk.NewCompositeStore(chunkStoreConfig, schemaConfig, storageClient)
```

Comment:
ISTM that you could have left the exported name as NewStore() to minimise impact elsewhere.

Reply:
Good idea, done.